### PR TITLE
include host_id in payload of a call that updates happ2host

### PR DESCRIPTION
**Pull request by peeech**
Not to be merged until tested with live resolver

### CODE REVIEW

File: src/types.rs [hunk 1]
```diff
@@ -20,6 +20,12 @@ pub struct Configuration {
     extra: HashMap<String, toml::Value>,
 }
 
+#[derive(Deserialize, Serialize, Debug)]
+struct Payload {
+    host_id: String,
+    happ_urls: Vec<String>,
```
Comment (alastairong) [inline src/types.rs]: Resolver expects `happ_ids` not `happ_url`

File: src/types.rs [hunk 2]
```diff
@@ -112,17 +118,21 @@ impl Configuration {
...
         let url = "https://resolver.holohost.net/update/addHost";
 
+        let host_id = crate::utils::get_host_id()?;
+
         let happ_urls = self
             .dnas
             .iter()
             .cloned()
             .filter(|dna| dna.holo_hosted)
             .filter_map(|dna| dna.happ_url)
```
Comment (alastairong) [inline src/types.rs]: Is this the right information? We are expecting the happ ID (hash) not a url / domain name.

Comment (peeech) [reply src/types.rs]: u r right

**Review by Br1ght0ne — APPROVED** [review]

**Review by alastairong — APPROVED** [review]
LGTM

**Pull request merged**